planetiler-core/src/main/java/com/onthegomap/planetiler/reader/osm/PolyFileReader.java

```diff
@@ -0,0 +1,82 @@
+package com.onthegomap.planetiler.reader.osm;
+
+import static com.onthegomap.planetiler.geo.GeoUtils.JTS_FACTORY;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.CoordinateXY;
+import org.locationtech.jts.geom.MultiPolygon;
+import org.locationtech.jts.geom.Polygon;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PolyFileReader {
```

Comment:
Can you add a quick blurb to the javadoc here, and maybe an `@see` link to the spec for it?